### ISSUE TITLE
completed tasks?

### ISSUE THREAD

**Issue by TimOgilvy** [darekkay/todoist-export#1]
Hi!

This thing is actually brilliant and I really appreciate it.  I want completed tasks too tho.  

Is it hard or easy to add this?  Not sure what the API is like yet. Happy to work on helping you do it if I can.

Thanks tho, this gadget saved my @ss.

**Comment by darekkay** [comment]
I'm using the Sync API v5.3, which returns all data with one call. It doesn't return completed tasks, though. You would need to traverse the projects manually.
The current [API v6](https://developer.todoist.com/#overview) was simplified and now it's obviously possible to get all completed tasks with one call. I will look into this.

**Comment by TimOgilvy** [comment]
Thanks for your response!  I ended up learning the API and building myself something on PHP to pull the whole lot out.  It's not too complex but it did take a bit of time to de-cypher the whole thing.

What I found most intriguing is that sub-projects actually don't have parents, they are simply indented. This somewhat reduces the usefulness of todoist for me. I'm not sure if the next version of todoist will address this.
